.github/workflows/on-push-branch.yml

```diff
@@ -56,6 +56,8 @@ jobs:
     secrets:
       CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
       CRN_SQUIDEX_CI_CLIENT_SECRET: ${{ secrets.CRN_SQUIDEX_CI_CLIENT_SECRET }}
+      CONTENTFUL_MANAGEMENT_TOKEN: ${{ secrets.CONTENTFUL_MANAGEMENT_TOKEN }}
```

Comment:
even though we're not going to run for environment `Development` or `Production` we still need `CONTENTFUL_MANAGEMENT_TOKEN` and `CRN_CONTENTFUL_ACCESS_TOKEN`  on `on-push-master` workflow since they're required in `reusable-test.yml`, right?

Reply:
Yes, probably.